### ISSUE TITLE
Please support zstd format `.zst`

### ISSUE THREAD

**Issue by Delta-in-hub** [wummel/patool#116]
```
> patool extract test.tar.zst
patool: Extracting test.tar.zst ...
patool error: error extracting test.tar.zst: unknown archive format for file `test.tar.zst'
```
patool 1.12

**Comment by vitlav** [comment]
Check this PR: https://github.com/wummel/patool/pull/95

**Comment by wummel** [comment]
this is fixed in 3156199475f4673eaf3d9e233cb3d37e1d913f6f
